usr.bin/localedef/bootstrap/ctype.h

```diff
@@ -44,13 +44,23 @@
  * However, only do so if they are not already define as a macro (as is the
  * case on Linux).
  */
-#ifndef isxdigit
 __BEGIN_DECLS
+#ifndef isalpha
 int	isalpha(int);
```

Comment:
quick question here: there's two ways to fix this: one is how you've done (and is likely fine for bootstrap), the other is to make these int (isalpha)(int); to prevent the macro expansion. Is there a reason to prefer one over the other?

Reply:
I guess you shouldn't have a prototype for something that's implemented as a macro, since it doesn't really exist